src/lib.nr

```diff
@@ -1,14 +1,13 @@
 mod mut_sparse_array;
 use dep::sort::sort_advanced;
 
-unconstrained fn __sort_field_as_u32(lhs: Field, rhs: Field) -> bool {
+unconstrained fn __sort_field_as_u32(lhs: u32, rhs: u32) -> bool {
     //  lhs.lt(rhs)
-    lhs as u32 < rhs as u32
+    lhs < rhs
 }
```

Comment:
This is now unnecessary

Reply:
Ah ok, I missed that this was being passes through into `noir_sort`. This can then be renamed as it's no longer sorting fields.